### PR TITLE
TRUNK-5956: Removed redundant Boolean literals

### PR DESCRIPTION
**Pull request by insookwa**
## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-5956

## Checklist: I completed these to help reviewers :)

- [ x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by sacull** [comment]
Please look here: https://wiki.openmrs.org/display/docs/Pull+Request+Tips

Commit names should be written like: "TRUNK-0000: Fix something".
Try to correct it.

**Comment by insookwa** [comment]
> Please look here: https://wiki.openmrs.org/display/docs/Pull+Request+Tips
> 
> Commit descriptions should be written like: "[TRUNK-0000](https://issues.openmrs.org/browse/TRUNK-0000): Fix something".
> Try to correct it.

Is that fine @sacull . Thanks for the advice

**Comment by sacull** [comment]
I meant this:
image

;)

File: api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java [hunk 1]
```diff
@@ -215,7 +215,7 @@ public Order getOrder(Integer orderId) throws DAOException {
 		}
         Criterion fulfillerStatusCriteria = null;
         if (searchCriteria.getIncludeNullFulfillerStatus() != null ) {
-            if (searchCriteria.getIncludeNullFulfillerStatus().booleanValue() == true ) {
+            if (searchCriteria.getIncludeNullFulfillerStatus().booleanValue() ) {
```
Comment (sacull) [inline api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java:218]: You forgot to delete the space in front of the last closing brace (the area marked with a red dot).
image

**Review by sacull — CHANGES_REQUESTED** [review]

File: api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java (hunk 1 above)
Comment (insookwa) [reply api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java:218]: oh okay let me change close it and do a git commit --amend for the message and git push --force for that space

**Review by sacull — APPROVED** [review]

**Comment by sacull** [comment]
It looks good to me. @dkayiwa please write do you require squashing these two commits?

**Comment by dkayiwa** [comment]
@sacull her is the answer: https://talk.openmrs.org/t/rfc-favour-not-squashing-commits-over-squashing-commits/28482

**Pull request merged**